src/test/jtx/TestHelpers.h

```diff
@@ -468,8 +476,7 @@ template <class Suite>
 void
 checkMetrics(
     Suite& test,
-    int line,
-    char const* file,
+    source_location const location,
```

Comment:
```suggestion
    source_location const location = source_location::current(),
```
This should simplify it even further.

Reply:
It does, but it will result in `Location: .../rippled/src/test/jtx/TestHelpers.h:489` where doing it like this results in `Location: .../rippled/src/test/app/Batch_test.cpp:3528`

Reply:
Not from my experience - it's from the caller. There's an example here: https://en.cppreference.com/w/cpp/utility/source_location